jest.config.js

```diff
@@ -9,12 +9,6 @@ module.exports = {
 		'@/([^\\.]*)$': '<rootDir>/src/$1',
 		'\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
 			'<rootDir>/__mocks__/fileMock.js',
-		// need to alias preact to itself in order to use the cjs version, same with testing-library
```

Comment:
Alias or not Jest does not want to use the CJS version no matter what I tried. Instead I've resorted to just using the ESM version and transforming it to CJS with `@swc/jest`.